acceptance-setup/__tests__/acceptance.js

```diff
@@ -72,7 +72,15 @@ beforeAll(done => {
         port: registryPort,
         baseUrl: registryUrl,
         env: {},
-        templates: components.map(component => component.template)
+        templates: components
+          .map(component => component.template)
+          .filter(template => {
+            // we need only the non-default templates here
```

Comment:
here is solved the mistery around duplicated externals on the oc client. The templates here need to be only the non-default ones.